crates/astria-composer/src/executor/mod.rs

```diff
@@ -276,6 +290,37 @@ impl Executor {
                     }
                 }
             }
+        };
+
+        match reason {
```

Comment:
The logs here are confusing - shouldn't they mention 1) that the service is starting its shutdown procedure, 2) providing the `reason`, 3) reporting what other steps are taken prior to shutdown, and finally 4) reporting that it is now shutting down?

Reply:
Makes sense. I ll structure the logs accordingly.